templates/conferences/show.html.twig

```diff
@@ -49,7 +49,11 @@
                             <i class="fa fa-map"></i>&nbsp;
                             {{ 'conferences.show.sidebar.location_title'|trans }}
                         </p>
-                        <p>{{ conference.location }}</p>
+                        {% if conference.online %}
+                            <p>Online</p>
```

Comment:
Missing translation

Reply:
You want me to translate 'Online' ? I guess I'm missing something here because to me it looks like it is common to all languages ?
Or maybe I'm wrong and this is standard only to latin languages

Reply:
No matter if it translates the same in all languages or not, we must centralize all UI texts in translations files (to make them reusable and updatable all at once).